examples/vision/oml_omniglot.py

```diff
@@ -89,31 +99,41 @@ def main(
         meta_train_accuracy = 0.0
         meta_valid_error = 0.0
         meta_valid_accuracy = 0.0
+
+        learner = maml.clone()
+        evaluation_data_all_tasks = []
+        evaluation_labels_all_tasks = []
+
         for task in range(meta_batch_size):
             # Compute meta-training loss
-            learner = maml.clone()
             batch = tasksets.train.sample()
-            evaluation_error, evaluation_accuracy = fast_adapt(batch,
-                                                               learner,
-                                                               loss,
-                                                               adaptation_steps,
-                                                               shots,
-                                                               ways,
-                                                               device)
-            evaluation_error.backward()
+            evaluation_error, evaluation_accuracy, evaluation_data, evaluation_labels = \
+                fast_adapt(batch,
+                          learner,
+                          loss,
+                          adaptation_steps,
+                          shots,
+                          ways,
+                          device,
+                          freeze_feature_extraction=True)
```

Comment:
The freeze feature extraction is what guarantees that $\theta$ is not updated in the inner loop